packages/core/ui-settings/core-ui-settings-server-internal/src/ui_settings_service.test.ts

```diff
@@ -95,6 +97,14 @@ describe('uiSettings', () => {
           `"uiSettings for the key [foo] has been already registered"`
         );
       });
+
+      it('throws if registers the same key twice to global settings', async () => {
```

Comment:
NIT:

Add `it('doesn't throw when registering a global and namespaced setting with the same name'`